doc/content/reference/configuration/network-server.md

```diff
@@ -4,12 +4,39 @@ description: ""
 weight: 4
 ---
 
+## General configuration
```

Comment:
Title Case (also below) and "options" instead of "configuration" for consistency with other files in this section. Also below.